pyiron_base/project/data.py

```diff
@@ -36,14 +36,16 @@ def __new__(cls, *args, **kwargs):
         object.__setattr__(instance, "_project", None)
         return instance
 
-    def __init__(self, *args, project=None, **kwargs):
+    def __init__(self, *args, project=None, lazy=True, **kwargs):
         """
         A data storage container which can store itself to/retrieve itself from file at the project level.
 
         Args:
             project (pyiron_base.Project): The project instance the storage is attached to.
         """
         super().__init__(*args, **kwargs)
+        # Projectdata should be lazy by default.
+        self._lazy = lazy
```

Comment:
```suggestion
        super().__init__(*args, lazy=lazy, **kwargs)
```

This is probably what @liamhuber meant.

Reply:
ok then, thanks

Reply:
so what happens now?
will the issue [#835](https://github.com/pyiron/pyiron_base/issues/835) be closed?